GitCommands/DiffMergeTools/VsDiffMerge.cs

```diff
@@ -32,11 +32,13 @@ private static string GetVsDiffMergePath()
                 return ExeName;
             }
 
+            // For 2017 (15.0) and later, Common7 is not installed by default but often included
```

Comment:
For 2017 (15.0) and later, VsDiffMerge is not installed by default but often included

Reply:
fixed